### ISSUE TITLE
The result of backtesting 30 years is different from 5 years?

### ISSUE THREAD

**Issue by jacktang** [cvxgrp/cvxportfolio#159]
Hello, I play the [etf demo](https://www.cvxportfolio.com/en/stable/examples/etfs.html) and change the `backtest_many` code as below (for 5 years time window) since my laptop is not very high performance.

```
ress = sim.backtest_many(
    [make_policy(*key) for key in keys], parallel=True, start_time=pd.Timestamp.today() - pd.Timedelta(f'{365.24*5}d'))
```
And the plots

![image](https://github.com/cvxgrp/cvxportfolio/assets/44341/92fa042f-79aa-4af6-806b-5ac996f8b7be)

And the original one for around 30 years time window

![image](https://github.com/cvxgrp/cvxportfolio/assets/44341/ba3ac35d-517f-4159-b95f-04d816f90c54)

The portfolio value trend is very similar, but the largest 7 assets are quite different. Especially for cash. Can you explain why it changes so much? Thanks!

**Comment by enzbus** [comment]
That's a good question, I hope it's not because Yahoo Finance data has changed in the meantime. Can you run the whole thing to try (it's monthly re-sampled, so it shouldn't be too bad). In general every back-test is path dependent so it will change based on the starting conditions, but this does seem like a large difference.

**Comment by jacktang** [comment]
@enzbus I run the demo with 30 years time window, and the result is almost the same with the example page, below is the `largest_growth_figure`

![image](https://github.com/cvxgrp/cvxportfolio/assets/44341/e362871b-645a-4f42-88a1-4b4f60df9d4c)

**Comment by enzbus** [comment]
Yeah, if read the code carefully you'll see that example actually runs many back-test sweeping over hyper-parameters, so each one will have different allocations.

**Comment by jacktang** [comment]
It is interesting :) I'd like to read the code deeply. BTW, is there any verbose option for debugging or output more detail logging?

**Comment by enzbus** [comment]
Yes, adding this two lines at the start should suffice:

```
import logging
logging.basicConfig(level='INFO')
```
